proxy-node-acceptance-tests/features/smoke/integration/integration.feature

```diff
@@ -0,0 +1,7 @@
+Feature: eidas-proxy-node-smoke-test-integration feature
+
+    Scenario: Integration proxy node happy path - LOA Substantial
+        Given the proxy node is sent a LOA 'Substantial' request
```

Comment:
Don't you need something here that suggests a stub, or anything that forces this to be integration rather than prod?

Reply:
So in the steps the 'Given the proxy node is sent a LOA 'Substantial'...' will `visit(ENV.fetch('STUB_CONNECTOR_URL') + loa_url)` and that STUB_CONNECTOR_URL is passed in the verify-terraform

Reply:
Ah, ok if that's _always_ a stub then perhaps it could be '..request from the stub connector' because then it's more clear?